sdk/java-sdk/src/test/scala/kalix/javasdk/impl/action/ActionHandlerSpec.scala

```diff
@@ -68,7 +70,8 @@ class ActionHandlerSpec
   private val anySupport = new AnySupport(Array(ActionspecApi.getDescriptor), this.getClass.getClassLoader)
 
   def create(handler: ActionRouter[_]): Actions = {
-    val service = new ActionService(_ => handler, serviceDescriptor, Array(), anySupport)
+    val actionFactory: ActionFactory = _ => handler
+    val service = new ActionService(actionFactory, serviceDescriptor, Array(), anySupport, None)
```

Comment:
For some reason, after adding the `Option[ActionOptions]`, the type inference stopped to work for `ActionFactory`. That's why I'm had to type it explicitly. 🤷